### DISCUSSION TITLE
Is "enter blocked junction" entirely pointless?

### DISCUSSION THREAD

**Discussion by originalfoo** [CitiesSkylinesMods/TMPE#1269] [Q&A]
Unless I'm missing something, "Enter blocked junction" seems to be a completely redundant feature, because it's handled (should be) much better by priority signs.

From priorty sign perspective:

* Priority = Enter blocked junction
* Yield = Slow down, only enter if safe to do so, otherwise stop
* Stop = Stop, wait for junction to clear before entering

From junction restriction perspective:

* allow 'enter blocked junction' = acts like priority sign
* deny 'enter blocked junction' = acts like yield sign (but would act like stop sign with higher simulation accuracy - see https://github.com/CitiesSkylinesMods/TMPE/issues/1229#issuecomment-995324724 )

The functionality overlap creates weird contradictions. Currently, I can set a `stop` sign (which means "stop and wait for junction to clear before entering") but simultaneously allow `enter blocked junction` which defeats purpose of the `stop` sign.

**Comment by kvakvs** [comment]
Sometimes cars will briefly pause on entirely random nodes, that helps in that scenario.
Also for roundabouts. The inner ring must flow at all times. Current behaviour of the traffic AI will pause on those nodes too if they're busy.

**Reply by originalfoo** [reply]
That's what priority signs are for?

If anything, `enter blocked junction` should act more like `box junction` and thus be a (future) feature of timed traffic lights.

My suggestion would be to remove 'enter blocked junction' from Junction Restrictions tool, and move Priority Signs in to Junction Restrictions. So now in JR you see the priority sign circle where the enter blocked junction used to be.

What triggered me thinking about this (again) was looking at translation for the Roundabout Policy `Yielding vehicles keep clear of blocked roundabout` and I'm like "hang on, isn't that just a stop sign?"
